deployments/kubernetes/install/scripts/install-cni.sh

```diff
@@ -155,14 +155,21 @@ CNI_CONF_FILE=/host/etc/cni/net.d/${CNI_CONF_NAME}
 if [ -e "/host/etc/cni/net.d/${CNI_CONF_NAME}" ]; then
     # This section overwrites an existing plugins list entry to for istio-cni
     CNI_TMP_CONF_DATA=$(cat ${TMP_CONF})
-    CNI_CONF_DATA=$(cat ${CNI_CONF_FILE} | jq 'del( .plugins[]? | select(.type == "istio-cni"))' | jq ".plugins += [${CNI_TMP_CONF_DATA}]")
+    CNI_CONF_DATA=$(cat ${CNI_CONF_FILE} | jq --argjson CNI_TMP_CONF_DATA "$CNI_TMP_CONF_DATA" -f /filter.jq)
     echo "${CNI_CONF_DATA}" > ${TMP_CONF}
 fi
 
 # Delete old CNI config files for upgrades.
 if [ "${CNI_CONF_NAME}" != "${CNI_OLD_CONF_NAME}" ]; then
     rm -f "/host/etc/cni/net.d/${CNI_OLD_CONF_NAME}"
 fi
+
+# If the old config filename ends with .conf, rename it to .conflist, because it has changed to be a list
+if [ "${CNI_CONF_NAME: -5}" == ".conf" ]; then
```

Comment:
This section is ending up creating another separate file and not removing the old file.  IMO, we should either just overwrite the old file or remove it and create the .conflist file.  If you put this `if` section prior to the section at line 162 it will `rm` the old file.